docs/infrastructure/deployment-targets/linux/index.md

```diff
@@ -20,14 +21,14 @@ The following platforms are explicitly supported (we run automated tests against
 - Ubuntu 18.04 LTS
 - Ubuntu 16.04 LTS
 - Redhat (RHEL) 7.2
-- Centos 7.6
+- Centos 7.7
```

Comment:
I've checked all those versions against the buildagentautomation repository. They look correct.